dev/index.html

```diff
@@ -53,7 +53,7 @@
                     <option value="HorizontalVertical">Horizontal-Vertical</option>
                     <option value="Horizontal">Horizontal</option>
                     <option value="Vertical">Vertical</option>
-                    <option value="Center" selected="yes">Center</option>
```

Comment:
**With `selected="yes"`**
![image](https://user-images.githubusercontent.com/4921914/122920416-aba5f280-d361-11eb-8f56-28bcbb0e30d1.png)

Reply:
I know but mutliple IDE hightlight me the error